engine/src/available-command.ts

```diff
@@ -523,10 +546,15 @@ export function possibleSpaceLostPlanet(engine: Engine, player: Player) {
       continue;
     }
 
+    const warnings: BuildWarning[] = [];
+    if (qicNeeded > 0 && p.hasActiveBooster(Resource.TemporaryRange)) {
+      warnings.push("range-booster-not-used");
+    }
+
     spaces.push({
-      building: Building.Mine,
```

Comment:
Why remove the building?

Reply:
it was not used IIUC

Reply:
I'm adding more type info as I change code - so this is how I found out.